content/en/apps/guides/data/analytics/couch2pg-to-cht-sync-migration.md

```diff
@@ -0,0 +1,31 @@
+- **DBT Modelling**: Avoid the temptation to model new DBT models after existing SQL views and tables. Instead, take the opportunity to re-evaluate the data needs and design new models that are more efficient and effective. Think of what data needs to be shown and how it should be shown in data visualization tools and use that to guide the design of the new models.
+<!-- TODO: Link to docs for testing DBT models once they are ready -->
+- **Testing**: After migrating, thoroughly test the new DBT models to ensure that they are working as expected.
+- **Feedback**: Provide any feedback and create issues for errors or bugs encountered in the [CHT Sync](https://github.com/medic/cht-sync) and [CHT Pipeline](https://github.com/medic/cht-pipeline/) repositories to improve the tools.
+
+## Migration Steps
+1. **Plan the migration**: Determine the scope of the migration, including the data sources, the data models, and the data transformations. Identify the existing SQL views, tables, and dashboards and assess what data you want to visualize. 
+1. **Set up CHT Sync**: Follow the instructions [to setup CHT Sync locally](({{< relref "apps/guides/data/analytics/setup" >}})).
+1. **Build DBT models**:  Use the [dedicated guidelines](({{< relref "apps/guides/data/analytics/building-dbt-models" >}})) to build DBT models for the data you want to visualize.
+1. 1. **Deploy CHT Sync**: Once the DBT models are tested locally and working as expected, deploy CHT Sync in production. Follow [the instructions to set up CHT Sync in production](({{< relref "apps/guides/data/analytics/production" >}})). It is recommended that CHT Sync be run in parallel with couch2pg during the migration process. This minimises disruption to users of the existing dashboards because they can continue to use the existing data while the new pipeline is being set up. It also makes it easier to compare the data from couch2pg when testing the new pipeline.
+1. **Create replica dashboards**: In the data visualization tool of your choice, create replica dashboards of your current setup and compare the data from the old and new pipelines.
+1. **Test and adjust the DBT models**: Test the DBT models to ensure they are working as expected and that the replica and initial dashboards match. Adjust the models to ensure they are accurate.
+1. **Optimize**: Once the DBT models are working as expected and the dashboards display the expected data, optimize the models to improve performance. This may involve restructuring the models, adding indexes, or making other adjustments to improve the speed and efficiency of the models. Having a look at the [DBT models guide](({{< relref "apps/guides/data/analytics/building-dbt-models" >}})) will help you understand how to optimize the models.
```

Comment:
```suggestion
1. **Optimize**: Once the dbt models are working as expected and the dashboards display the expected data, optimize the models to improve performance. This may involve restructuring the models, adding indexes, or making other adjustments to improve the speed and efficiency of the models. Having a look at the [dbt models guide](({{< relref "apps/guides/data/analytics/building-dbt-models" >}})) will help you understand how to optimize the models.
```